### DISCUSSION TITLE
Class error

### DISCUSSION THREAD

**Discussion by andree-2** [flet-dev/flet#1963] [Q&A]
### Question

Hello everyone, I'm having a problem starting Flet, I simply created the main function within a class, but unfortunately the content of “self.rail” is not seen even if self.rail and a simple ft.Text(“hello”) but if I don't use the variable but write everything inside the main function starts, how can I solve it?

### Code sample

```python
def main(self,page = ft.Page):
        page.bgcolor = ft.colors.WHITE
        page.add(self,ft.Row(
                [
                    self.rail,
                    ft.VerticalDivider(width=1),
                    ft.Column([ ft.Text("Body!")], alignment=ft.MainAxisAlignment.START, expand=True),
                ],
                expand=True,
            )
        )      
        page.update(self,)
```

### Error message

```bash
"AttributeError: the object 'Page' does not have the attribute 'rail'."
```

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Please Build up a proper repro code sample so we can better assist you.
